src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-contact-list-interface.cypress.spec.js

```diff
@@ -13,40 +13,46 @@ describe('Secure Messaging Contact list', () => {
   it('verify base web-elements', () => {
     ContactListPage.verifyHeaders();
     ContactListPage.verifyAllCheckboxes(true);
-    ContactListPage.clickSelectAllCheckBox();
+    ContactListPage.selectAllCheckBox();
     ContactListPage.verifyAllCheckboxes(false);
     ContactListPage.verifyButtons();
 
     cy.injectAxe();
     cy.axeCheck(AXE_CONTEXT);
   });
 
-  it('verify saving changes alert', () => {
-    ContactListPage.clickSelectAllCheckBox();
-    ContactListPage.clickSaveModalCancelButton();
+  it('verify saving changes alerts', () => {
+    ContactListPage.selectAllCheckBox();
+    ContactListPage.clickCancelButton();
     ContactListPage.verifySaveAlertHeader();
     ContactListPage.verifyButtons();
     ContactListPage.closeSaveModal();
 
     ContactListPage.clickBackToInbox();
     ContactListPage.verifySaveAlertHeader();
     ContactListPage.verifyButtons();
+    ContactListPage.closeSaveModal();
+
+    ContactListPage.clickSaveAndExitButton();
+    ContactListPage.verifyEmptyContactListAlert();
+
+    ContactListPage.selectCheckBox(`100`);
 
     ContactListPage.clickSaveAndExitButton();
     ContactListPage.verifyContactListSavedAlert();
 
+    cy.wait('@savedList')
+      .its('request.body')
+      .then(req => {
+        const obj = req.updatedTriageTeams.find(el => el.name.includes(`100`));
+        expect(obj.preferredTeam).to.eq(true);
+      });
+
```

Comment:
I would suggest adding a coverage for a scenario with 2 or more facilities

Reply:
done